reflex-algos/src/main/python/mlops/parallelm/mlops/time_capture/parsers.py

```diff
@@ -175,23 +175,15 @@ def _parse_mlapp(self, file_path):
         """
         with open(file_path, 'r') as f:
             reader1 = f.read()
-        reader1 = reader1.replace('wfNodes', ',wfNodes')
-        reader1 = reader1.replace('GroupID', ',GroupID')
-        reader1 = reader1.replace('Mode', ',Mode')
-        reader1 = reader1.replace('Type', ',Type')
-        reader1 = reader1.replace(',', '","')
-        reader1 = reader1.replace(':', '":"')
-        reader1 = reader1.replace('" [', '[{"')
-        reader1 = reader1.replace(']"', '"}]')
-        reader1 = reader1.replace(' ', '')
-        reader1 = reader1.replace(',"PipelineID', '},{"PipelineID')
-        nd1 = '{"' + reader1 + '"}'
-        kd = json.loads(nd1)
+        kd = json.loads(reader1)
```

Comment:
mlops has some code to build info info from jsons - anything we can reuse?

Reply:
Parsing here is simple. I change the code to use the mlops constants. At a certain point we can further align with the elements that the  REST command returns to maybe use the _detect_ion_structure() and get all the mlops APIs active. Is there a preference to do it now? Currently I just wanted to get the pipeline orientation in the graphs.